### PR TITLE
Airship documentation update

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Updated Airship documentation to remove the app secret dependency.

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

## Notion ticket link

> [Airship Use API Key](https://www.notion.so/rudderstacks/Airship-Use-API-Key-5422f0a60c034590a0ebdff5d7186cb4)

### CODE REVIEW

File: destinations/marketing/airship.mdx [hunk 1]
```diff
@@ -45,19 +45,18 @@ To successfully configure Airship as a destination, you will need to configure t
 
 <img src="../../assets/event-stream-destinations/airship-connection-settings.png" alt="Airship connection settings" />
 
-- **API Key**: Enter your Airship token which is required by RudderStack to communicate with your Airship project.
+- **API Key**: Enter your Airship token required by RudderStack to communicate with your Airship project.
```
Comment (utsabc) [inline destinations/marketing/airship.mdx:48]: App key is only used for track calls maybe we can mention that using callout?
@ujjwal-ab ?

Comment (ameypv-rudder) [reply destinations/marketing/airship.mdx:48]: Makes sense. I will add this info.

Comment (utsabc) [reply destinations/marketing/airship.mdx:48]: API key is used for all calls , only track requires app key cc@ujjwal-ab

Comment (ujjwal-ab) [reply destinations/marketing/airship.mdx:48]: Yes, App key is required in track call only.

Comment (ameypv-rudder) [reply destinations/marketing/airship.mdx]: I've made the changes. Please review. @utsabc @ujjwal-ab

File: destinations/marketing/airship.mdx [hunk 2]
```diff
@@ -45,19 +45,18 @@ To successfully configure Airship as a destination, you will need to configure t
 
 <img src="../../assets/event-stream-destinations/airship-connection-settings.png" alt="Airship connection settings" />
 
-- **API Key**: Enter your Airship token which is required by RudderStack to communicate with your Airship project.
+- **API Key**: Enter your Airship token which is required by RudderStack to communicate with your Airship project and authenticate all the supported event calls, i.e.[`identify`](#identify), [`track`](#track), and [`group`](#group).
```
Comment (ishachopra0401) [inline destinations/marketing/airship.mdx:48]:
```suggestion
- **API Key**: Enter your Airship token which is required by RudderStack to communicate with your Airship project and authenticate all the supported event calls, that is, [`identify`](#identify), [`track`](#track), and [`group`](#group).
```

**Pull request merged**